### ISSUE TITLE
List share modal window hangs browser.

### ISSUE THREAD

**Issue by DrKley** [go-vikunja/frontend#60]
![image](https://user-images.githubusercontent.com/2788877/173404816-68ead583-0033-42ce-a965-03c1f5c72417.png)

Browser: 
Chrome Version 102.0.5005.115 (Official Build) (64-bit)
(Also downloaded and tried on the latest version of firefox)

**Comment by kolaente** [comment]
Can you reproduce this on [try](https://try.vikunja.io)? It works there for me on Firefox/Chromium and Linux (Nixos).

Which OS are you using?

**Comment by DrKley** [comment]
> Can you reproduce this on [try](https://try.vikunja.io)? It works there for me on Firefox/Chromium and Linux (Nixos).
> 
> Which OS are you using?

I´m glad you asked, I just tried on the demo site and found out that in order to reproduce the bug you'll need to first share any List with a group or user, it will actually record it to the DB and freeze the browser, if you try to access the share modal after that it will freeze again.

Not an expert but seems to be a problem when the frontend fetches the current added users/groups on the modal window as the issue does not affect the lists when there's no one added. I've also tried with a stable branch instance of the frontend and that modal window works perfectly it just lacks some of the functionality

I´m using Chrome on Windows as the client and my instance is hosted on a docker container on Ubuntu.
